Jenkinsfile_CNP

```diff
@@ -152,7 +152,7 @@ withPipeline(type, product, component) {
     before('smoketest:preview') {
       sh """
           eval \$(./bin/variables/load-preview-environment-variables.sh ${CHANGE_ID})
-          ./bin/import-ccd-definition.sh "-e *-prod.json,*HNL-nonprod.json,*CUI-nonprod.json,*-shuttered.json" ${ccddefbranch}
+          ./bin/import-ccd-definition.sh "-e *-prod.json,*HNL-nonprod.json,*-shuttered.json" ${ccddefbranch}
```

Comment:
require the same changes in  before('smoketest:aat') { otherwise master pipeline will fail